modules/ROOT/pages/mergetags.adoc

```diff
@@ -0,0 +1,55 @@
+= Merge Tags plugin
+:navtitle: Merge Tags
+:description: Enables Merge Tags functionality in the {productname} editor
+:keywords: merge, tags, tag, mergetags, mergetag, mergetags_list, mergetags_prefix, mergetags_suffix, mail merge, personalization token
+:pluginname: Merge Tags
+:plugincode: mergetags
+:pluginminimumplan: tiertwo
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a merge tag (also known as a personalization token, or a mail merge field) from a drop-down list when a specified prefix is typed.
+
+Once the merge tag is inserted, the plugin leaves
+
+* a non-editable variable wrapped in a prefix; and
+* a suffix making the tag easily identifiable.
```

Comment:
```suggestion
Once the merge tag is inserted, the plugin leaves a non-editable variable wrapped with a prefix and suffix, making it easily identifiable.
```

Reply:
We use `a merge tag` in the description then `One the merge tag` here. So it `the` vs `a` not sure if it matters and what the language rules are here in English.

```suggestion
Suggested change 
Once a merge tag is inserted, the plugin leaves a non-editable variable wrapped with a prefix and suffix, making it easily identifiable.
```